nixos/modules/services/web-servers/keter/default.nix

```diff
@@ -0,0 +1,163 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.services.keter;
+in
+{
+  meta = {
+    maintainers = with lib.maintainers; [ jappie ];
+  };
+
+  options.services.keter = {
+    enable = lib.mkEnableOption ''keter, a web app deployment manager.
+        Note that this module only support loading of webapps:
+        Keep an old app running and swap the ports when the new one is booted.
```

Comment:
Why does this get extra indent? This will also get rendered to a manpage where space is not infinite. :P

Reply:
I deleted the extra indent, I didn't realize this was used in manpages, that's pretty cool.

Reply:
`man configuration.nix`